docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -1,17 +1,23 @@
+
+* *Data Distribution:* Each shard contains a portion of the data from the index. When you add more nodes to your cluster, {es} will spread the shards across the nodes, balancing the workload between them.
+* *Replication:* Shards can have replicas which are copies of the original shard. Replicas ensure data availability and improve search performance by allowing multiple nodes to handle requests for that shard.
+* *Parallel Processing:* Shards enable {es} to process queries in parallel across nodes, making searches faster and more efficient.
+
+By effectively using shards, {es} can scale horizontally and provide fault tolerance, ensuring your data is distributed and queries are processed efficiently.
+
 [[size-your-shards]]
-== Size your shards
-
-Each index in {es} is divided into one or more shards, each of which may be
-replicated across multiple nodes to protect against hardware failures. If you
-are using <<data-streams>> then each data stream is backed by a sequence of
-indices. There is a limit to the amount of data you can store on a single node
-so you can increase the capacity of your cluster by adding nodes and increasing
-the number of indices and shards to match. However, each index and shard has
-some overhead and if you divide your data across too many shards then the
-overhead can become overwhelming. A cluster with too many indices or shards is
-said to suffer from _oversharding_. An oversharded cluster will be less
-efficient at responding to searches and in extreme cases it may even become
-unstable.
+=== Size your shards
+
+Proper shard sizing is crucial for maintaining the performance and stability of an {es} cluster. _Oversharding_ occurs when data is distributed across an excessive number of shards, which can degrade search performance and make the cluster unstable. Conversely, very large shards may slow down search operations and prolong recovery times after failures. To strike the right balance, the <<shard-size-recommendation,general guidelines>> are to aim for shard sizes between 10GB and 50GB, keeping the per-shard document count below 200 million. To ensure that each node is working optimally, it's important to distribute shards evenly across nodes. Having uneven shard distribution will make some nodes work harder than others leading to performance degradation and potential instability. 
+
+If you are using <<data-streams>> then each data stream is backed by a sequence of indices which are all broken into their own shards. Proper allocation of shards for each backing index is crucial to maintain performance and stability. 
```

Comment:
```suggestion
If you are using <<data-streams>> then each data stream is backed by a sequence of indices, each index potentially having multiple shards. 
```

(deleting the last sentence since the distribution of shards was mentioned exactly above at the end of the previous paragraph)